expectMessage/utils/colorizeLog.ts

```diff
@@ -46,12 +47,42 @@ const createStyle = (cat: any, styleName?: string): string => {
   return styleName ? `\x1b[${styles[cat][styleName.toLowerCase()]}m` : "";
 };
 
-export const colorize = (string: string, customStyle: TerminalCustomStyle = {}): string => {
+const createDefault = (
+  string: string,
+  fg: string,
+  bg: string,
+  effects = ""
+): string => {
   return [
-    createStyle("fg", customStyle.fg),
-    createStyle("bg", customStyle.bg),
-    createStyle("effects", customStyle.effects),
+    createStyle("fg", fg),
+    createStyle("bg", bg),
+    createStyle("effects", effects),
     string.toString().replace(/\s*$/, ""),
-    customStyle.fg || customStyle.bg || customStyle.effects ? createStyle("effects", "reset") : "",
+    fg || bg || effects ? createStyle("effects", "reset") : "",
   ].join("");
 };
+
+export const colorize = (
+  string: string,
+  customStyle: TerminalCustomStyle = {}
+): string => {
+  if (typeof customStyle === "string" || typeof customStyle === "number") {
+    if (customStyle === "code red" || customStyle === 3) {
+      return createDefault(string, "yellow", "red", "blink");
+    } else if (customStyle === "warning" || customStyle === 2) {
+      return createDefault(string, "red", "", "underscore");
+    } else {
+      return createDefault(string, "cyan", "", "");
+    }
+  } else {
```

Comment:
After thinking about that, it's not the place to do this.

The colorize function musn't have a default for warning or other things because it's **just** a function that colorise strings.
It's the `expectMessage` role to call colorize with a "style" object:
If we give `expectMessage` a style object, it should call colorize with the object otherwise, if we call expectMessage with a string (like `"warning"`) in place of a customStyle object, the `expectMessage` function should call a `warning` function which call `colorize` with `{bg: "red", effects: "bright"}`.

Reply:
Fenn suggested to add the `warning` or `error` functions (or stuff like that) into the colorize package. I'm fine with both.
@Fuzznimp what do you prefer?

Reply:
For now is present 
- error
- warning
- log
Added as a string after the custom message. If I understood correctly you prefer having it under a function than a parameter?
I can get rid of the numbers as well btw